server/src/rest/structs.rs

```diff
@@ -5,6 +5,7 @@
 //!
 //! Types here are different from the openapi types.
 #![allow(dead_code)]
+#![allow(missing_docs)]
```

Comment:
do you need these? resolve if possible without using dead_code or missing_docs

Reply:
Removed dead_code, still need missing docs since many struct fields are self-evident enough to not have docs.

[d227142](https://github.com/Arrow-air/svc-assets/pull/18/commits/d2271428712237b35575f5be1116dae5f0b0dd87)